pkg/console/controllers/managedclusters/controller.go

```diff
@@ -226,12 +226,37 @@ func (c *ManagedClusterController) SyncManagedClusterList(ctx context.Context) (
 			continue
 		}
 
+		// Get Claim values in a map
+		claims := util.ClaimsMap(&managedCluster)
+
+		// Omit clusters that have the following product name
+		omittedProductCluster := []string{"AKS", "EKS", "IKS", "ROKS"}
+		if ProductCheckForClusterList(claims, omittedProductCluster) {
+			klog.V(4).Infof("Skipping managed cluster %q, product %q is unsupported", clusterName, claims["product.open-cluster-management.io"])
+			continue
+		}
+
+		// Omit any clusters with version less than ocp 4.0
+		if VersionCheckForClusterList(claims) {
+			klog.V(4).Infof("Skipping managed cluster %q with version %q, versions prior to openshift 4.0 not supported", clusterName, claims["version.openshift.io"])
+			continue
+		}
 		valid = append(valid, managedCluster)
 	}
 
 	return valid, "", nil
 }
 
+func VersionCheckForClusterList(claims map[string]string) bool {
+	s, ok := claims["version.openshift.io"]
+	return ok && s < "4.0"
```

Comment:
Again, I would rather use a proper solution for comparing version, when plane string coparision.
This package we already vendor in the operator https://github.com/openshift/console-operator/blob/master/go.mod#L7

Reply:
ok, switched to Compare function